src/utils/collection.ts

```diff
@@ -191,12 +191,11 @@ export function generateCollectionInsert(collection: ResolvedCollection, data: P
   })
 
   const valuesHash = hash(values)
-  if (opts.hashColumn !== false) {
-    values.push(`'${valuesHash}'`)
-  }
```

Comment:
We need to add back this

Reply:
A system with no hashColumn would not benefit from this improvement. 
I will set it up.